mrjob/sim.py

```diff
@@ -647,7 +648,7 @@ def _chmod_u_rx(path, recursive=False):
             for file_name in file_names:
                 _chmod_u_rx(join(dir_name, file_name))
     else:
-        if hasattr(os, 'chmod'):  # only available on Unix, Windows
+        if hasattr(os, 'chmod') and platform.system() != "Windows":  # only available on Unix
```

Comment:
This comment isn't really honest. Something like, "this causes rmtree() to fail on Windows; see #1847" would be better (and it's perfectly fine to devote a whole line to the comment).